backend/src/test/java/de/neuefische/backend/controller/JobControllerTest.java

```diff
@@ -0,0 +1,68 @@
+package de.neuefische.backend.controller;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import de.neuefische.backend.model.Job;
+import de.neuefische.backend.repo.JobRepo;
+import de.neuefische.backend.service.GenerateUUIDService;
+import de.neuefische.backend.service.JobService;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+
+
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class JobControllerTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private JobService jobService;
+
+    @Autowired
+    private JobRepo jobRepo;
+
+    @Autowired
+    private GenerateUUIDService generateUUIDService;
```

Comment:
no need for that anymore.
use lombok.config
and the tests will be much slimmer